subcommand/common/common.go

```diff
@@ -30,3 +32,16 @@ func Logger(level string) (hclog.Logger, error) {
 		Output: os.Stderr,
 	}), nil
 }
+
+// ValidatePort converts flags representing ports into integer and validates
+// that it's in the port range.
+func ValidatePort(flagName, flagValue string) error {
+	port, err := strconv.Atoi(flagValue)
+	if err != nil {
+		return errors.New(fmt.Sprintf("%s value of %s is not a valid integer.", flagName, flagValue))
+	}
+	if port <= 1023 || port > 65353 {
```

Comment:
```suggestion
	if port < 1024 || port > 65353 {
```

Reply:
As Kyle mentioned, is it `65535` and not `65353`? Could you add a one-liner here and on `handler.go:700` as to why this range.

Reply:
Yup!!